cmd/crank/xpkg/dep.go

```diff
@@ -0,0 +1,217 @@
+/*
+Copyright 2023 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package xpkg
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/alecthomas/kong"
+	"github.com/pterm/pterm"
+	"github.com/spf13/afero"
+
+	"github.com/crossplane/crossplane-runtime/pkg/errors"
+
+	"github.com/crossplane/crossplane/apis/pkg/v1beta1"
+	"github.com/crossplane/crossplane/internal/xpkg/v2"
+	"github.com/crossplane/crossplane/internal/xpkg/v2/dep"
+	"github.com/crossplane/crossplane/internal/xpkg/v2/dep/cache"
+	"github.com/crossplane/crossplane/internal/xpkg/v2/dep/manager"
+	"github.com/crossplane/crossplane/internal/xpkg/v2/dep/resolver/image"
+	"github.com/crossplane/crossplane/internal/xpkg/v2/workspace"
+)
+
+const (
+	errMetaFileNotFound = "crossplane.yaml file not found in current directory"
+)
+
+// AfterApply constructs and binds context to any subcommands
+// that have Run() methods that receive it.
+func (c *depCmd) AfterApply(kongCtx *kong.Context, p pterm.TextPrinter) error {
+	kongCtx.Bind(pterm.DefaultBulletList.WithWriter(kongCtx.Stdout))
```

Comment:
@lsviben did you want to inherit pterm for output? We originally added it up for the upbound installation and it seems to have propagated to a few other areas (like this subcommand). I mostly ask since it's unclear if the new crossplane cli wants to use pterm output or if it wants to have the option to take a different path.

Reply:
Also it should be pretty easy to remove pterm from these subcommands if you want to decrease the scope of what is being pulled over 👍.

Reply:
When I saw pterm there and checked it out I liked how you could setup the style. Just added the skeleton of it so we dont copy Upbound style. But maybe its a good idea to leave it out and add it properly in a separate PR.

Reply:
>But maybe its a good idea to leave it out and add it properly in a separate PR

Sounds great to me. I think if we leave it in here, it's less obvious that it's being added to the repo and why.